### ISSUE TITLE
Unknown checkbox in tools bar, no accessible name

### ISSUE THREAD

**Issue by jimmydarko** [edrlab/thorium-reader#2711]
In the tools bar on the right side of the boookmark checkbox there is a checkbox that has no accessible name.
I use a screen reader (JAWS) because I am visually impaired.
My screen reader just says "Checkbox not checked, dialogue".
